.cirrus.yml

```diff
@@ -139,17 +134,9 @@ task:
   use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_PR == ''
   container:
     dockerfile: .ci/Dockerfile
-    cpu: 8
-    memory: 16G
-  env:
-    INTEGRATION_TEST_PATH: "./packages/integration_test"
-  upgrade_script:
-    - flutter channel stable
-    - flutter upgrade
-    - flutter channel master
-    - flutter upgrade
-    - flutter config --enable-linux-desktop
-    - git fetch origin master
+    cpu: 4
+    memory: 12G
```

Comment:
Might have to play with this if you start seeing OOM failures.

Reply:
Yep, I figured this was safe since I wasn't seeing it going above 10 in general, but we can certainly adjust it back up if necessary.